Makefile

```diff
@@ -6,6 +6,8 @@ all:
 check:
 	python3 -m pyflakes tasks tasks/container/webhook
 	python3 -m pycodestyle --max-line-length=120 --ignore=E722 tasks tasks/container/webhook
+	if command -v mypy >/dev/null && pip show types-PyYAML >/dev/null; then mypy test; else echo "SKIP: mypy or types-PyYAML not installed"; fi
```

Comment:
Yeah, that's something I'd like to do as a follow-up, or prerequisite. It involves a tasks container refresh to add it there, as the integration test runs itself inside of the tasks container, and would fail without it. I sent #609, if/when that lands I can drop the `pip show` part.

Reply:
OK, `# type: ignore[import-untyped]` DTRT, and still typechecks if the types are actually installed. So I went with that.